docfx.json

```diff
@@ -95,63 +95,63 @@
              "xml/**/*": "None"
       },
       "ms.service": {
-        "api/Microsoft.Identitymodel*.yml": "active-directory",
-        "api/Microsoft.Identity*.yml": "active-directory",
+        "api/DeviceExplorer*.yml": "iot-hub",
+        "api/KeyVault.TestFramework*.yml": "key-vault",
+        "api/Microsoft.ApplicationInsights*.yml": "application-insights",
+        "api/Microsoft.Azure.Batch*.yml": "batch",
```

Comment:
ah! so much better. so now I can compare with that folder and see missing paths like Microsoft.Azure.ApplicationInsights 😄 
Do you wanna double check that list?

Reply:
@mariraw Done. I think the whole thing looks good, you?